resources/zib/zib-NursingIntervention.xml

```diff
@@ -0,0 +1,437 @@
+  <name value="ZibNursingIntervention" />
+  <title value="zib NursingIntervention" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A nursing intervention is the care and/or treatment carried out by a nurse based on an expert opinion and clinical knowledge. It is part of the nursing process; interventions are determined as a result of indexed healthcare problems (nursing diagnoses) and based on treatment goals. Following the NEN-EN-ISO 18104 norm, it is possible to structure nursing interventions (or actions) in a hierarchy, where one intervention can be part of another intervention.&#xD;&#xA;&#xD;&#xA;In a home care situation some actions can also be performed by the patient or a caregiver after a relevant instruction." />
+  <purpose value="This CarePlan resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) NursingIntervention-v3.2(2020EN)](https://zibs.nl/wiki/NursingIntervention-v3.2(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-nursingintervention-v3.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/NursingIntervention-v3.2(2020EN)" />
+    <name value="zib NursingIntervention-v3.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="CarePlan" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/CarePlan" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="CarePlan">
+      <path value="CarePlan" />
+      <short value="NursingIntervention" />
+      <alias value="VerpleegkundigeInterventie" />
+      <constraint>
+        <key value="zib-NursingIntervention-01" />
+        <severity value="error" />
+        <human value="The start and/or end date of a repeating schedule should be communicated in CarePlan.encounter as well." />
+        <expression value="activity.detail.scheduledTiming.repeat.boundsPeriod.exists() implies (period = activity.detail.scheduledTiming.repeat.boundsPeriod)" />
+        <source value="http://nictiz.nl/fhir/StructureDefinition/zib-NursingIntervention" />
+      </constraint>
+      <mapping>
+        <identity value="zib-nursingintervention-v3.2-2020EN" />
+        <map value="NL-CM:14.2.1" />
+        <comment value="NursingIntervention" />
+      </mapping>
+    </element>
+    <element id="CarePlan.category">

```

Comment:
Shouldn't there be an implicit mapping on .status based on period.start and .end (ProcedureStartDateTime and ProcedureEndDateTime)?

Reply:
And can we say something about a .intent matching the zib? 'plan' perhaps?

Reply:
Good point, I'll look into this.

Reply:
I've added comments on `.status` and `.intent` for guidance on this topic. I'm reluctant to fix `.intent` on plan because information standard might provide extra context that affects this, so I only wrote a comment.